### DISCUSSION TITLE
A lot of error messages on welcome page

### DISCUSSION THREAD

**Discussion by poste0** [apache/superset#19948] [Ideas]
There are a lot of error messages shown on the welcome page if a user doesn't have permissions to read Dashboards, Charts, Saved Queries or Recent activity. This happens In case of redirect from a forbidden page as well. This doesn't look well when a user sees an error messages stack at the time.

#### How to reproduce the bug

1. Create a role without permissions to read Dashboards, Charts, Saved Queries, Recent Activity
2. Set this role only to a user
3. Login as this user
4. There are a lot of error messages on welcome page

### Expected results

Don't show a stack of messages to the user as it doesn't look well

### Actual results

There are a lot of messages

#### Screenshots

![image](https://user-images.githubusercontent.com/26258521/166662882-66737bc6-6d7c-4716-af31-e08a1dd8ab0c.png)

### Environment

(please complete the following information):

- browser type and version: Chrome 100.0.4896.127 and Microsoft Edge 101.0.1210.32
- superset version: The latest version built from sources on master branch
- python version: 3.8.10
- node.js version: 16.9.1
- any feature flags active: Checked on default superset from master branch

**Comment by srinify** [comment]
Hey there, moving to Discussions so people can chime on how to make the experience better! But this isn't a "bug" per say in the strict sense unfortunately because it's working as intended ;)
